ksql-engine/src/test/java/io/confluent/ksql/security/DefaultStatementPermissionsValidatorTest.java

```diff
@@ -13,10 +13,12 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package io.confluent.ksql.engine;
+package io.confluent.ksql.security;
 
 import static org.mockito.Mockito.when;
 
+import io.confluent.ksql.engine.KsqlEngine;
+import io.confluent.ksql.engine.KsqlEngineTestUtil;
```

Comment:
What are the new uses for these two imports in this test? I'm not sure I understand why these were added.

Reply:
Wait never mind I see, it got moved out of the engine package

Reply:
Never mind, I see, since this was moved out of engine, those need to be imported.